core/src/test/java/org/mskcc/cbio/portal/scripts/TestIntegrationTest.java

```diff
@@ -176,7 +205,76 @@ public void testLoadStudyEs0() {
             }
             assertEquals(50, countGte2Lt3);
     		
+            //===== check cancer_type
+            List<DBCancerType> cancerTypes = apiService.getCancerTypes(Arrays.asList("brca-es0"));
+            assertEquals(1, cancerTypes.size());
+            assertEquals("Breast Invasive Carcinoma", cancerTypes.get(0).name);
+            
+            //===== check fusion data
+            //TODO - depends on fix for #1102 and #1314
+            
+            
+            //===== check gistic data
+            //servlet uses this query:
+            ArrayList<Gistic> gistics = DaoGistic.getAllGisticByCancerStudyId(cancerStudy.getInternalId());
+            assertEquals(14, gistics.size());
+            Gistic gisticChr10 = null, gisticChr20 = null;
+            for (Gistic gistic: gistics) {
+                if (gistic.getChromosome() == 20) {
+                    //assert not yet set:
+                    assertEquals(null, gisticChr20);
+                    gisticChr20 = gistic;
+                }
+                else if (gistic.getChromosome() == 10) {
+                    //assert not yet set:
+                    assertEquals(null, gisticChr10);
+                    gisticChr10 = gistic;
+                }
+            }
+            assertEquals(8, gisticChr10.getGenes_in_ROI().size());
```

Comment:
ROI is a non-obvious acronym to a new developer. Maybe change this function to spell out what this means (RegionOfInterest?)  Also, google java style specifies that method names should be lower camel case : https://google.github.io/styleguide/javaguide.html#s5.2.3-method-names. Refactoring this may be outside the scope of this PR .. but maybe it can be done without too much pain.